src/main/resources/static/css/app.css

```diff
@@ -1,143 +0,0 @@
-body {
```

Comment:
You won't find it O_O but that's because I removed it at one point to get the admin theme up and running. Let me know what you'd like back.